packages/@ember/-internals/glimmer/lib/modifiers/internal.ts

```diff
@@ -78,6 +78,10 @@ export class InternalModifierManager
     return null;
   }
 
+  getDebugInstance(state: InternalModifierState): unknown {
+    return state;
+  }
```

Comment:
I think just declare the class `abstract` and leave this out. We have to decide case-by-case what we want to expose here. And so far this is only used by `{{action}}` anyway

Reply:
TS says its required, -- I'll return null by default tho

Reply:
Yeah, that's why I said *declare the class `abstract`*... when you _concretely_ implement/extend an abstract class, you are required to implement all the abstract members (which is the point of abstract classes). 

_However,_ you can also abstractly extend a class (`abstract class Foo extends AnotherAbstractClass`), in which case, you are allowed to leave out any abstract members, essentially deferring/propagating the requirement to your children